app/front/scripts/controllers/main.js

```diff
@@ -69,7 +73,24 @@ angular.module('Application')
       function initRegular() {
         $q(LoginService.tryGetToken())
           .then(function(token) {
-            return $q(osViewerService.loadDataPackages(token));
+            var packageId = null;
+            var userId = token ? LoginService.getUserId() : null;
+            if (!token) {
```

Comment:
Okay, so the issue here is that we will query only for packages owned by the user (without the package id).
Does `getInitialState` only look within the available packages, or does it perform its own search query?